R/link.R

```diff
@@ -0,0 +1,32 @@
+library(stringr)
+library(readr)
+
+#' Link the data set to the reference data.
+#'
+#' @param input_path Source Parquet file path.
+#' @param output_path Target Parquet file path.
+#' @param patient_path String. Path to the patient ID bridge file.
+#'
+#' @export
+#'
+link <- function(input_path, output_path, patient_path, demographics_path, deaths_path) {
```

Comment:
As no value is defined for `output_path` it is then an expected argument that must be provided (I think).

Based on `normalizePath(file.path(output_path), mustWork = FALSE)` you could define a default variable but common practice (with my Python head on, not sure if this is true for R and how it uses positional arguments) it would need moving to the end of the function definition.

```suggestion
link <- function(input_path, patient_path, demographics_path, deaths_path, , output_path="./") {
```